# Sensor data from robot
bool touch_detected
float32 distance  # cm
float32 temperature  # Celsius
time timestamp
